### PR TITLE
#784 - Nits to fix and refactoring

### PR DESCRIPTION
**Pull request by shiranr**
# Pull Request Template

## What are you trying to address

#784 - Nits to fix and refactoring

## Description of new changes

- Movement of text from one place to another
- Rephrasing 

https://github.com/microsoft/code-with-engineering-playbook/issues/784

### CODE REVIEW

File: docs/code-reviews/faq.md [hunk 1]
```diff
@@ -4,18 +4,34 @@ This is a list of questions / frequently occurring issues when working with code
...
 | Source code focused | Intended to enhance and enable code reviews. Includes both source code but can have a broader scope (e.g., docs, integration tests, compiles) |
 | Intended for **early feedback** before submitting a PR | Not intended for **early feedback**. Created when author is ready to merge |
 | Usually a synchronous review with faster feedback cycles (draft PRs as an exception). Examples: scheduled meetings, over-the-shoulder review, pair programming | Usually a tool assisted asynchronous review but can be elevated to a synchronous meeting when needed |
 
-## We experience very large PRs, how can we fix this?
+## Why do we need code reviews?
+
+Our peer code reviews are structured around best practices, to find specific kinds of errors. Much like you would still run a linter over mobbed code, you would still ask someone to make the last pass to make sure the code conforms to expected standards and avoids common pitfalls.
+
+## How to perform a review?
```
Comment (tompaana) [inline docs/code-reviews/faq.md:20]: I feel like we are losing a bit of information and context here. Perhaps we should revert these changes a little. I suggest moving this section to its old place (not including the one above ["Why do we need code reviews?"]) and combine the old content under one heading e.g., "We pair or mob. How should this reflect in our code reviews?"

**Review by tompaana — CHANGES_REQUESTED** [review]
Very nice, thank you! I added couple of suggestions and ideas.

File: docs/code-reviews/README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ Developers working on [CSE](../CSE.md) projects should conduct peer code reviews
...
 
-Code reviews is a way to have a conversation about the code where participants will:
+Code review is a way to have a conversation about the code where participants will:
 
-- Improve code quality by identifying and removing defects before they can be introduced into shared code branches.
-- Grow by learning from each other about unfamiliar design patterns or languages among other topics, and even break some bad habits.
-- Develop a shared understanding of the project's code.
+- Improve code quality - identifying and removing defects before they can be introduced into shared code branches.
+- Learn and grow - by having others review the code, we get exposed to unfamiliar design patterns or languages among other topics, and even break some bad habits.
+- Shared understanding - between the developers over the project's code.
```
[the remark this answers is not in the document]
Comment (shiranr) [reply docs/code-reviews/README.md:11]: done.

File: docs/code-reviews/faq.md (hunk 1 above)
Comment (shiranr) [reply docs/code-reviews/faq.md]: Fixed.

**Review by TessFerrandez — APPROVED** [review]
